mule-apikit-module/src/main/java/org/mule/module/apikit/FlowFinder.java

```diff
@@ -92,6 +91,11 @@ protected void initializeRestFlowMap()
         }
     }
 
+    private List<Flow> getFlows() {
```

Comment:
Can we unified this usage with org.mule.module.apikit.metadata.ApikitElementIdentifiers?
We should move ApikitElementIdentifiers to apikit-module.